src/MEDS_DEV/models/meds_tab/tiny/model.yaml

```
@@ -1,3 +1,14 @@
+metadata:
+  description: >-
+    A tiny, low-cost XGBoost baseline using the MEDS-Tab framework. TODO(@Oufattole): Add more details here.
+  links: ["https://arxiv.org/abs/2411.00200"]
```

```suggestion
  links:
    - https://arxiv.org/abs/2411.00200
```

(To make it consistent in style with MIMIC-IV/dataset.yaml file)